quipucords/scanner/satellite/six.py

```diff
@@ -675,15 +716,22 @@ def hosts_facts(self, manager_interrupt):
 
                     if manager_interrupt.value == ScanJob.JOB_TERMINATE_PAUSE:
                         raise SatellitePauseException()
-                    host_params = [(host.get(ID), host.get(NAME),
-                                    ssl_cert_verify, defined_host, port, user,
-                                    password)
-                                   for host in chunk]
-                    results = pool.starmap(self.host_details, host_params)
-                    for result in results:
-                        if result is not None:
-                            self.record_inspect_result(
-                                result.get('name'),
-                                result.get('details'),
-                                result.get('status'))
+                    host_params = self.prepare_host(chunk)
+                    results = pool.starmap(get_http_responses,
+                                           host_params)
+                    for each in results:
+                        if each:
+                            result = post_processing_results(
+                                each[0],
```

Comment:
This probably would be more readable with a dict instead of a list.  That would remove the each[0], each[1], ... and replace with each['unique_name'].